ex2_services/Services/source/adcs/adcs_service.c

```diff
@@ -1531,6 +1531,13 @@ SAT_returnState adcs_service_app(csp_packet_t *packet) {
  * @return None
  */
 void adcs_service(void *param) {
+    int32_t iErr = red_mkdir("adcs");
+    if (iErr == -1 && red_errno != 17) {
+        sys_log(ERROR, "Unexpected error %d from creating adcs directory", red_errno);
+    } else {
+        sys_log(INFO, "Successfully created adcs directory");
+    }
+
```

Comment:
Just wondering why do we move adcs directory creation from below to start of function?

Reply:
https://github.com/AlbertaSat/ex2_obc_software/pull/287/commits/d1ec26862ca5864651783f5b5ee739ace2c2df94
The description of this commit